node/res/turing.json

```diff
@@ -15,7 +19,7 @@
     "tokenDecimals": 10,
     "tokenSymbol": "TUR"
   },
-  "relay_chain": "ksmcc3",
```

Comment:
I'm not sure what `ksmcc3` is but setting the relay chain as kusama allows the node to infer the relay chain spec without explicitly passing a file.

Reply:
we got it from here: https://github.com/paritytech/polkadot/blob/master/node/service/res/kusama.json

Reply:
Interesting. I'm 99% sure, `kusama` is what we actually want here, to be able to defer to polkadot for the chain spec.

Reply:
if it works w/ our chain, that's fine. We'll hear about issues from the collators